internal/services/mssql/mssql_server_resource_test.go

```diff
@@ -177,6 +177,28 @@ func TestAccMsSqlServer_azureadAdmin(t *testing.T) {
 	})
 }
 
+func TestAccMsSqlServer_passwordUpdate(t *testing.T) {
```

Comment:
Might prefer rename to be more precise (see comments below)

Reply:
Maybe the `TestAccMsSqlServer_azureadAdminWithAADAuthOnlyWithPasswordUpdate` would be more appropriate.